apps/teams-test-app/e2e-test-data/profile.json

```diff
@@ -23,9 +23,9 @@
           "width": 0,
           "height": 0
         },
-        "triggerType": "MouseClick"
+        "triggerType": "MouseHover"
```

Comment:
I am so confused how could it work for web if you didn't make any change in Hub SDK side but simply replace MouseClick to MouseHover? It is not an input so I assume it should have a failure here.

Reply:
Maybe due to this is the default input for the test app... for example, in iOS, we also don't use this default value for testing. I'd image the default values are mostly used by devs when they are doing the manual testing (so they can just use default and don't have to manually find the input). Does it sound reasonable?

Reply:
No change would be needed in the hub-sdk because the hub-sdk is just taking this entire payload it receives as part of the call and displaying the whole thing it in an alert. In this case, this test is only used in the web, and it's passing up MouseHover as the trigger type, and so the test is expecting that to show up in an alert.

The default value in ProfileAPIs.tsx is only added if a user presses the "Default" button in the test app, it's intended to make manual testing using the test app more straightforward, it can be tricky to figure out what format each call wants added to the text box without it. Making a change there should have no effect on the E2E tests.